.github/workflows/maven.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       uses: actions/cache@v3
       with:
         path: ~/.m2/repository
-        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml', '**/*.target') }}
```

Comment:
@laeubi this should be useful for Tycho as well, shouldn't it?

Reply:
sure, it should not harm much.